var/spack/repos/builtin/packages/cube/package.py

```diff
@@ -36,6 +36,11 @@ class Cube(Package):
     homepage = "http://www.scalasca.org/software/cube-4.x/download.html"
     url = "http://apps.fz-juelich.de/scalasca/releases/cube/4.2/dist/cube-4.2.3.tar.gz"
 
+    # Some versions fail with parallel builds/installs
+    parallel = False
```

Comment:
Yes, that's what you would have to do. If you happen to know which version first fixed the parallel build, we could set it up to so that it builds in parallel for newer versions and builds in serial for older versions. It would always have to install in serial.

If this sounds like too much work, globally setting `parallel = False` is fine.